src/Asset/Parcel/Parcel.model.js

```diff
@@ -33,10 +33,6 @@ export class Parcel extends Model {
     return splitCoordinate(id)
   }
 
-  static findByIds(ids) {
```

Comment:
Why we are removing it?

Reply:
Not used anywhere and was a security concern, ids was being interpolated and could be SQL injected